src/code/z_quake.c

```diff
@@ -4,9 +4,18 @@
 QuakeRequest sQuakeRequest[4];
 s16 D_80126250 = 1;
 s16 sQuakeRequestCount = 0;
+
+// clang-format off
 s16 (*sQuakeCallbacks[])(QuakeRequest*, ShakeInfo*) = {
-    NULL, Quake_Callback1, Quake_Callback2, Quake_Callback3, Quake_Callback4, Quake_Callback5, Quake_Callback6,
+    NULL,
+    Quake_Callback1,
+    Quake_Callback2,
+    Quake_Callback3,
+    Quake_Callback4,
+    Quake_Callback5,
+    Quake_Callback6,
 };
+// clang-format on
```

Comment:
I would argue this isn't worth a clang-format disable. If we start trying to keep arrays vertical like this we will need 100's of clang-format disables everywhere.